pyxem/signals/diffraction2d.py

```diff
@@ -563,22 +565,23 @@ def find_peaks(self, method, *args, **kwargs):
 
         Parameters
         ---------
-        method : str
+        method : string
```

Comment:
str is preferred, eg) https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Ok, will do a find an replace to fix this throughout the code base.